.orchestra/ci/mass-testing-weekly/entrypoint.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -euo pipefail
+SCRIPT_DIR=$(realpath "$(dirname "${BASH_SOURCE[0]}")")
+
+TIMESTAMP=$(date +%s)
+
+# Install dependencies and the orchestra tool
+sudo apt-get update
+sudo apt-get install -y less nano vim inkscape bc
+
+cd "$SCRIPT_DIR/../../.."
+.orchestra/ci/install-dependencies.sh
+pip3 -q install --user "git+https://github.com/revng/revng-orchestra.git@master"
+export PATH="$HOME/.local/bin:$PATH"
+
+# Copy user_options.yml and update
+cp -a "$SCRIPT_DIR/user_options.yml" .orchestra/config/user_options.yml
+orc update --no-config
+
+# Run the progress poller, which will print the progress of mass-testing every
+# hour
+"$SCRIPT_DIR/progress_poller.sh" &
+POLLER_PID=$!
+
+# Run the actual mass-testing
+RC=0
+orc --quiet install mass-testing || RC=$?
```

Comment:
Who is pushing the results to S3?
Below I think we're pushing to mass.rev.ng.

Reply:
> Who is pushing the results to S3?

That's taken care by the `orc install mass-testing`, in the sense that the install script pushes to S3

>  Below I think we're pushing to mass.rev.ng.

Exactly

Reply:
It seems a bit arbitrary to have the `mass-testing` component to push things to S3 but another script push to mass.rev.ng.

1. Can we move the push to S3 here?
2. Can you summarize what we push to S3?

Reply:
> It seems a bit arbitrary to have the mass-testing component to push things to S3 but another script push to mass.rev.ng.

`mass-testing` already fiddles with S3 since it has to download the binaries etc. so it made sense to also make it push to S3

1. Sure it can be done
2. Just `regression.autogenerated.csv`, that's the only file that's ever pushed to S3

Reply:
What if we put `regression.autogenerated.csv` on mass.rev.ng and avoid having S3 write credentials?

Reply:
It can be done, but then we rely on `mass.rev.ng` working for regression to work, I'm not sure it's a great dependency
Do note that the credentials used by mass-testing allow to *exclusively* write `regression.autogenerated.csv`. It cannot (over)write any other file

Reply:
We really need to unify the infrastructure. We have too many things spread around.

Anyway, let's keep things as they are for now.